src/client/query.rs

```diff
@@ -1,3 +1,66 @@
+use crate::model::crypto::Signature;
+use std::time::SystemTime;
+
+/// Represents client's query to `Iroha` peer.
+#[derive(serde::Serialize, serde::Deserialize)]
+pub struct Query {
+    timestamp: u128,
+    signature: Signature,
```

Comment:
signature should be optional because for permissionless ledgers like Sora network, there isn't a need to check the signature

Reply:
fixed